src/main/java/software/bernie/techarium/tile/base/TechariumTileBase.java

```diff
@@ -0,0 +1,140 @@
+package software.bernie.techarium.tile.base;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.network.NetworkManager;
+import net.minecraft.network.play.server.SUpdateTileEntityPacket;
+import net.minecraft.tileentity.ITickableTileEntity;
+import net.minecraft.tileentity.TileEntityType;
+import net.minecraft.util.Direction;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.util.LazyOptional;
+import net.minecraftforge.energy.CapabilityEnergy;
+import net.minecraftforge.fluids.capability.CapabilityFluidHandler;
+import net.minecraftforge.items.CapabilityItemHandler;
+import org.jetbrains.annotations.NotNull;
+import software.bernie.techarium.trait.behaviour.IHasBehaviour;
+import software.bernie.techarium.trait.tile.TileBehaviour;
+import software.bernie.techarium.trait.tile.TileTraits;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Optional;
+
+import static software.bernie.techarium.util.StaticHandler.getSideFromDirection;
+
+public abstract class TechariumTileBase extends MachineTileBase implements ITickableTileEntity, IHasBehaviour {
```

Comment:
This class doesn't make any sense. This stuff should all be in either MachineTileBase or TechariumTile